geoutils/georaster.py

```diff
@@ -526,6 +527,96 @@ def shift(self, xoff, yoff):
                                                        d, dy, ymax + yoff)})
         self._update(metadata=meta)
 
+    def set_ndv(self, ndv, update_array=False):
+        """
+        Set new nodata values for bands (and possibly update arrays)
+
+        :param ndv: nodata values
+        :type ndv: list or int or float
+        :param update_array: change the existing nodata in array
+        :type update_array: bool
+        """
+
+        if not (isinstance(ndv, list) or isinstance(ndv, int) or isinstance(ndv, float)):
```

Comment:
A tuple would be fine too. I think now most people test this by comparing if the instance is a collections.Iterable: https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable

Reply:
Great tip. So basically collections.Iterable includes list, tuples, and arrays? (learning how to do things properly sure takes time :D)

Reply:
Yes, I just found about it recently. Not sure what else is included in these Iterable, but worth testing with tuple, list and arrays.